### PR TITLE
FSM v1 develop merge commit with code reivew comments

### PR DESCRIPTION
**Pull request by phanikiran-wtt-egov**
FSM v1 develop merge commit with code reivew comments

### CODE REVIEW

File: vendor/src/main/java/org/egov/vendor/web/model/vehicle/VehicleResponse.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+@Getter
+@NoArgsConstructor
+@Setter
+@ToString
+@Builder
+public class VehicleResponse {
+
+	@JsonProperty("responseInfo")
+	private ResponseInfo responseInfo = null;
+
```
Comment (talele08) [inline vendor/src/main/java/org/egov/vendor/web/model/vehicle/VehicleResponse.java:35]: add @Valid

File: vehicle/src/main/java/org/egov/vehicle/trip/web/model/VehicleTripResponse.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@Getter
+@Setter
+public class VehicleTripResponse {
+
+	@JsonProperty("responseInfo")
+	private ResponseInfo responseInfo = null;
+
```
Comment (talele08) [inline vehicle/src/main/java/org/egov/vehicle/trip/web/model/VehicleTripResponse.java:31]: Add @Valid annotation

File: vehicle/src/main/java/org/egov/vehicle/trip/web/model/VehicleTrip.java [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+  @JsonProperty("driverId")
+  private String driverId = null;
+  
+
+  @NotNull
+  @JsonProperty("vehicle")
+  private Vehicle vehicle;
+  
+
+  @JsonProperty("vehicleId")
```
Comment (talele08) [inline vehicle/src/main/java/org/egov/vehicle/trip/web/model/VehicleTrip.java:64]: add @size on all string fields and @Valid on all Class objects in all models

File: vendor/src/main/java/org/egov/vendor/repository/querybuilder/VendorQueryBuilder.java [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+	private static final String DRIVER_VEHICLE_QUERY = "SELECT %s FROM %s where %s = ?";
+	private static final String VEHICLE_EXISTS = "SELECT vendor_id FROM eg_vendor_vehicle where vechile_id IN ";
+
+	private static final String DRIVER_ID = "driver_id";
+	private static final String VEHICLE_ID = "vechile_id";
+	private static final String VENDOR_ID = "vendor_id";
+	private static final String VENDOR_DRIVER = "eg_vendor_driver";
+	private static final String VENDOR_VEHICLE = "eg_vendor_vehicle";
+
+	public String getDriverSearchQuery() {
```
Comment (talele08) [inline vendor/src/main/java/org/egov/vendor/repository/querybuilder/VendorQueryBuilder.java:40]: use preparedStatement

File: vendor/src/main/java/org/egov/vendor/web/model/Vendor.java [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+
+	@JsonProperty("address")
+	private Address address = null;
+
+	@JsonProperty("owner")
+	private User owner = null;
+
+	@JsonProperty("vehicles")
+	@Valid
+	private List<Vehicle> vehicles = new ArrayList<Vehicle>();
```
Comment (talele08) [inline vendor/src/main/java/org/egov/vendor/web/model/Vendor.java:54]: Plead add @size and @Valid annotations

**Review by talele08 — CHANGES_REQUESTED** [review]

File: vendor/src/main/java/org/egov/vendor/web/model/Vendor.java (hunk 5 above)
Comment (quasim-wtt-egov) [reply vendor/src/main/java/org/egov/vendor/web/model/Vendor.java:58]: Done

File: vehicle/src/main/java/org/egov/vehicle/trip/web/model/VehicleTrip.java (hunk 3 above)
Comment (quasim-wtt-egov) [reply vehicle/src/main/java/org/egov/vehicle/trip/web/model/VehicleTrip.java:70]: Done

File: vehicle/src/main/java/org/egov/vehicle/trip/web/model/VehicleTripResponse.java (hunk 2 above)
Comment (quasim-wtt-egov) [reply vehicle/src/main/java/org/egov/vehicle/trip/web/model/VehicleTripResponse.java:33]: Done

File: vendor/src/main/java/org/egov/vendor/web/model/vehicle/VehicleResponse.java (hunk 1 above)
Comment (quasim-wtt-egov) [reply vendor/src/main/java/org/egov/vendor/web/model/vehicle/VehicleResponse.java:35]: Done

**Review by talele08 — APPROVED** [review]

**Pull request merged**